.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: deploy:heroku:deploy-app
+
+on:
+  push:
+    branches:
+      - 'release/**'
+
+jobs:
+  deploy_app:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git-checkout
+        uses: actions/checkout@v2
+
+      - name: heroku-deploy-app
+        uses: akhileshns/heroku-deploy@v3.12.12
+        env:
+          HD_NODE_CONFIG_ENV: "qa"
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_email: ${{secrets.HEROKU_EMAIL}}
+          heroku_app_name: "node-ts-boilerplate-qa"
+          dontautocreate: true
+
+      - name: heroku-add-to-pipeline
+        run: heroku pipelines:add backend-pipeline --app=node-ts-boilerplate-qa --stage=staging || true
```

Comment:
I think the app will already be in a pipeline, no need to add it again.

Reply:
@akashlakhera  sir, actually there may be a case in which an app exist with this name on Heroku but is not added to the pipeline, so just to make sure that we add the app with this name to the pipeline I have added this step to add the app to pipeline.

Reply:
What will be the case? Can you give an example? We'll always manually add the add to a pipeline.

Reply:
I added it just to be on safe side, no issues I will remove this add to pipeline step.